### DISCUSSION TITLE
[@types/node] globalThis is broken after newest release

### DISCUSSION THREAD

**Discussion by quantizor** [DefinitelyTyped/DefinitelyTyped#70945] [Issues with a @types package]
I think this commit might have been the cause @Renegade334 https://github.com/DefinitelyTyped/DefinitelyTyped/commit/2a130227d4205668060c04bea3a327d970c736a0

If I upgrade from 20.16.11 to 20.16.12 any `global.*` usage is broken

![Screenshot 2024-10-18 at 12 10 34 PM](https://github.com/user-attachments/assets/5feba687-a03d-417d-ba24-b37766316e22)

Related: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/69452

[a comment by an automated account is not shown]

**Comment by Renegade334** [comment]
I can't trivially reproduce this. Does `tsc` also report the error?

If so, could you provide your TS version, tsconfig.json, `npm ls`, and a minimal repro?
